### PR TITLE
Add organization setting

### LINKED ISSUE #1: Add a setting to support customizing GitHub Org

### PR DESCRIPTION
**Pull request by Nick-NCSU**
Closes #1

### CODE REVIEW

File: src/UserSetting.tsx [hunk 1]
```diff
@@ -134,7 +135,29 @@ const UserSetting: React.FC<{
...
           </div>
-          <FormLabel>Fetch Pull Requests for User</FormLabel>
+          <FormLabel>Reviewer Username</FormLabel>
+          <FormControl
+            type="text"
+            placeholder="Enter reviewer username"
+            value={userName}
+            onChange={(e) => {
+              setUserName(e.target.value);
+              localStorage.setItem('userName', e.target.value);
```
Comment (Nick-NCSU) [inline src/UserSetting.tsx]: Do we need this here if the handleClose also sets it in localStorage?

Comment (i5o) [reply src/UserSetting.tsx]: Please don't set this here, look at how we do it in the other element that has this placeholder: `placeholder="Enter username to fetch pull requests"`

File: src/UserSetting.css
Comment (i5o) [inline src/UserSetting.css:1]: Please use this instead:

https://react-bootstrap.netlify.app/docs/forms/validation/

**Review by i5o — CHANGES_REQUESTED** [review]
Thanks for your contribution, overall they look great.
Bootstrap has styles for the required behavior

**Review by i5o — APPROVED** [review]

**Pull request merged**